site/content/en/blog/releases/1.27.0-rc.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "1.27.0 - Release Candidate"
+linkTitle: "1.27.0-rc"
+date: "2022-10-19"
+---
+This is the 1.27.0-rc release of Agones.
+
+This release brings us support for go v 1.19.1, so if you are installing this release, remember to upgrade your version.
+
+This release has lots of important changes including:
+- Enable image streaming everywhere by default 
+- An example of setting autoscaler behavior in a Fleet. 
+- Log cleanup for pods not yet running
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.27.0-rc">README</a> for details on features, installation and usage.
```

Comment:
Without `data-proofer-ignore` this is going to fail. See the template: https://raw.githubusercontent.com/googleforgames/agones/main/docs/governance/templates/release.md